api/lib/source/cosmosV0-source.js

```diff
@@ -181,7 +184,7 @@ class CosmosV0API extends RESTDataSource {
     ).amount
   }
 
-  async getAllValidators(height) {
+  async getAllValidators(height, profile = false) {
```

Comment:
i see what you're doing here but i don't really understand why it's necessary to exclude the profiles. why is the `getAllValidators` call coupled with the call for individual profiles?

Reply:
there was repeated data in profile that is already present in validator.Makes sense to have these two tied together to have all validator related data in one place. Validators profile should also get stored in store, so why not just store everything together under validator (once someone queries for the validator profile of a particular validator)?

So, summing up, two reasons:
- not repeat data (we'd need to query for validators the same than in getAllValidators to get `name` and `website`)
- cleanliness storing data, make it easy to find & available

Reply:
ok thanks for the explanation.